localization/languages/uk.json

```diff
@@ -0,0 +1,184 @@
+        "serverNotFoundSubtitle": "Min не зміг знайти цей веб-сайт.",
+        "archiveSearchAction": "Пошук на archive.org",
+        "sslErrorTitle": "Сайт недоступний",
+        "sslErrorMessage": "Min не зміг безпечно з'єднатися з цим сайтом.",
+        "dnsErrorTitle": "Сайт не знайдено",
+        "dnsErrorMessage": "Сталася помилка DNS.",
+        "offlineErrorTitle": "Не вдалося встановити з'єднання з Інтернетом",
+        "offlineErrorMessage": "Підключіться до Інтернету та повторіть спробу.",
+        "genericConnectionFail": "Min не вдалося підключитися до веб-сайту.",
+        "sslTimeErrorMessage": "Мін не може надійно підключитися до цього веб-сайту. Переконайтеся, що годинник комп'ютера встановлено правильно.",
+        "addressInvalidTitle": "Недійсна адреса.",
+        "genericError": "Виникла помилка",
+        /* pages/phishing/index.html */
+        "phishingErrorTitle": "Цей сайт може вам зашкодити.",
+        "phishingErrorMessage": "Цей веб-сайт може намагатися викрасти вашу особисту інформацію, наприклад, паролі чи банківську інформацію.",
+        "phishingErrorVisitAnyway": "Все одно відвідати сайт",
+        "phishingErrorLeave": "Залишити цей сайт",
+        /* multiple instances alert */
+        "multipleInstancesErrorMessage": "Виникла помилка. Закрийте інші відкриті екземпляри та перезапустіть Min.",
+        /* pages/sessionRestoreError/index.html */
+        "sessionRestoreErrorTitle": "Виникла помилка",
+        "sessionRestoreErrorExplanation": "Ваші збережені вкладки не можуть бути відновлені правильно.",
+        "sessionRestoreErrorBackupInfo": "Ми зберегли резервну копію даних у цьому місці: %l.", //%l will be replaced with a path to a file
+        "sessionRestoreErrorLinkInfo": {
+            "unsafeHTML": "Якщо ця помилка продовжує з'являтися, будь ласка, повідомте про неї <a href=\"https://github.com/minbrowser/min\" target=\"_blank\">сюди</a>."
+        },
+        /* pages/settings/index.html */
+        "settingsPreferencesHeading": "Налаштування",
+        "settingsRestartRequired": "Щоб застосувати ці зміни, потрібно перезапустити програму.",
+        "settingsPrivacyHeading": "Блокування вмісту",
+        "settingsContentBlockingLevel0": "Дозволити всю рекламу та стеження",
+        "settingsContentBlockingLevel1": "Блокувати сторонню рекламу та стеження",
+        "settingsContentBlockingLevel2": "Блокувати всю рекламу та стеження",
+        "settingsContentBlockingExceptions": "Дозволити рекламу на цих сайтах:",
+        "settingsBlockScriptsToggle": "Блокувати скрипти",
+        "settingsBlockImagesToggle": "Блокувати зображення",
+        "settingsEasyListCredit": {
+            "unsafeHTML": "На основі EasyList і EasyPrivacy, без правил приховування елементів. Створений <a target=\"_blank\" href=\"https://easylist.to/\">Автори EasyList</a> - <a target=\"_blank\" href=\"https://creativecommons.org/licenses/by-sa/3.0/legalcode\">переглянути ліцензію</a>."
```

Comment:
The format of this string was changed slightly a couple of weeks ago to this:
```
Based on EasyList and EasyPrivacy, created by <a target=\"_blank\" href=\"https://easylist.to/\">The EasyList authors</a> (<a target=\"_blank\" href=\"https://creativecommons.org/licenses/by-sa/3.0/legalcode\">view license</a>).
```

Would you mind updating your translation to match?